test/common/router/router_ratelimit_test.cc

```diff
@@ -18,152 +18,66 @@ namespace Router {
 TEST(BadRateLimitConfiguration, MissingActions) {
   std::string json = R"EOF(
 {
-  "virtual_hosts": [
```

Comment:
You still need some test that tests that the main config properly loads an embedded rate limit. Do you still have at least one test that loads a full config?